### PR TITLE
udis86: move library into main package

### PR DESCRIPTION
**Pull request by Johnnynator**
otherwise we would have a cyclic dep udis86->udis86-devel->udis86

### CODE REVIEW

File: srcpkgs/udis86/template [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 version=1.7.2
-revision=3
+revision=4
 build_style=gnu-configure
-hostmakedepends="python"
 configure_args="--enable-shared"
+hostmakedepends="python"
 short_desc="Disassembler Library for x86 and x86-64"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
 license="BSD"
```
Comment (maxice8) [inline srcpkgs/udis86/template]: spdx short-identifier

Comment (Johnnynator) [reply srcpkgs/udis86/template]: We should at a SPDX License list to xlint to catch these. I rarely check the License of old packages.

Comment (maxice8) [reply srcpkgs/udis86/template]: The SPDX license list is massive

Comment (Johnnynator) [reply srcpkgs/udis86/template]: Yep, I'm aware of that. Alpine is afaik linting agianst the full list.

**Pull request merged**